vignettes/osmdata.Rmd

```diff
@@ -49,11 +49,11 @@ An `osmdata` query consists of an initial `opq()` query and one or more features
 q <- opq (bbox="greater london uk")
 q <- add_feature (q, key="highway", value="motorway")
 ```
-The corresponding data may then be extracted as `R Spatial` objects with,
+The corresponding data may then be extracted as a `Spatial` object defined by the `sp` package, with,
 ```{r osmdata_sp}
 osmdata_sp (q)
 ```
-or an `R Sinple Features` objects with
+or an `R Simple Features` objects with
```

Comment:
@mpadge I think it would be better to show the user first how to download the raw data onto the hd before converting it - more bandwidth efficient and a chance to show the `request -> dl -> conversion` steps in process. Happy to do this but cannot remember the function to simple dl raw OSM data.